pkg/apis/eks.cattle.io/v1/types.go

```diff
@@ -67,9 +67,9 @@ type EKSClusterConfigStatus struct {
 type NodeGroup struct {
 	Gpu                  *bool              `json:"gpu"`
 	ImageID              *string            `json:"imageId"`
-	NodegroupName        *string            `json:"nodegroupName" norman:"required"`
+	NodegroupName        *string            `json:"nodegroupName" norman:"required" wrangler:"required"`
```

Comment:
I haven't seen wrangler required before. Is that needed for codegen ?

Reply:
Yes. The codegen generates the CRD from the type. It looks for different struct tags and adds information to the CRD based on the tags. In this case, this adds `nodegroupName` to the `required` field for the `NodeGroups` (also in the diff).

Reply:
Really? I thought it recognized the norman tags?

Reply:
Running `go generate` with just the norman tags did not produce the `required` field.